tools/Google.Cloud.Tools.ReleaseManager/UpdateHistoryCommand.cs

```diff
@@ -17,6 +17,7 @@
 using LibGit2Sharp;
 using System;
 using System.Collections.Generic;
+using System.Diagnostics;
```

Comment:
Is this needed?

Reply:
Nope, sorry - that was left over from using Stopwatch. Removed.